staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -96,36 +109,45 @@ func isInterfaceUp(intf *net.Interface) bool {
 	return false
 }
 
-//getFinalIP method receives all the IP addrs of a Interface
-//and returns a nil if the address is Loopback, Ipv6, link-local or nil.
-//It returns a valid IPv4 if an Ipv4 address is found in the array.
-func getFinalIP(addrs []net.Addr) (net.IP, error) {
+func isLoopbackOrPointToPoint(intf *net.Interface) bool {
+	return intf.Flags&(net.FlagLoopback|net.FlagPointToPoint) != 0
+}
+
+func inFamily(ip net.IP, expectedFamily AddressFamily) bool {
```

Comment:
Looks like inFamily() and memberOf() are duplicates. Will remove duplication in part 3 commit set, rather than here and rebase again.